ta/ta.c

```diff
@@ -18,6 +18,15 @@
 #include <stdio.h>
 #include <assert.h>
 
+#ifdef __SANITIZE_ADDRESS__
+__attribute__((visibility("default")))
```

Comment:
Perhaps also make it weak:

```suggestion
__attribute__((visibility("default"), weak))
```

to allow libmpv users to overwrite it if they want to (?)

Reply:
ok